cardinal/t_gameSystems_test.go

```diff
@@ -624,6 +684,122 @@ func TestGetPlayersPresence_OtherPlayerAround(t *testing.T) {
 
 // #endregion Look System Test
 
+// #region Inventory System Test
+func TestInventory_WithItem(t *testing.T) {
```

Comment:
the test should test the inventory table as well as the output

Reply:
This test test if the player picks the ball and then tests if it is actually in the inventory. It tests the output which is a string